packages/vue/src/components/molecules/SfRange/SfRange.vue

```diff
@@ -0,0 +1,169 @@
+<template>
+  <div
+    ref="range"
+    v-focus
+    class="sf-range"
+    v-bind="$attrs"
+    :disabled="disabled"
+  ></div>
+</template>
+<script>
+import noUiSlider from "nouislider";
+import focus from "../../../utilities/directives";
+export default {
+  name: "SfRange",
+  directives: {
+    focus,
+  },
+  props: {
+    min: {
+      type: Number,
+      default: 0,
+    },
+    max: {
+      type: Number,
+      default: 0,
+    },
+    valueMin: {
+      type: Number,
+      default: 0,
+    },
+    valueMax: {
+      type: [Number, String],
+      default: 1,
+    },
+    step: {
+      type: Number,
+      default: 1,
+    },
+    label: {
+      type: String,
+      default: "",
+    },
+    disabled: {
+      type: Boolean,
+      default: true,
+    },
+    /*
+     * Horizontal orientation if true, vertical orientation if false
+     */
+    orientation: {
+      type: Boolean,
+      default: true,
+    },
+    /*
+     * Ltr orientation if true, rtl orientation if false
+     */
+    direction: {
+      type: Boolean,
+      default: true,
+    },
+    tooltips: {
+      type: Boolean,
+      default: false,
+    },
+  },
+  data() {
+    return {
+      config: {},
+      rangeOrientation: "horizontal",
+      rangeDirection: "ltr",
+    };
+  },
+  watch: {
+    min: {
+      handler(value) {
+        if (this.$refs.range) {
+          return this.updateOptions({
+            range: {
+              min: value,
+              max: this.max,
```

Comment:
Shouldn't a condition be checked here as in `mounted`?

Reply:
Yes you are right :)